### PR TITLE
Shubham

### PR DESCRIPTION
**Pull request by Shubhambal**
hi please could you review my changes ?

### CODE REVIEW

File: src/main/java/com/emart/controller/AdminController.java [hunk 1]
```diff
@@ -73,18 +74,24 @@ public ResponseEntity<Admin> getAdminById(@PathVariable int id) {
 	 */
 	@PostMapping("/create")
 	public ResponseEntity<Admin> createAdmin(@RequestBody Admin admin) {
```
Comment (sumukh-gn) [inline src/main/java/com/emart/controller/AdminController.java:76]: please add comments on what this method is performing

File: src/test/java/com/emart/AdminControllerTest.java [hunk 2]
```diff
@@ -22,71 +24,95 @@ public static void setup() {
...
                 .when()
-                .delete("/admins/delete/" + adminId)
+                .delete("/admins/" + adminId)
                 .then()
                 .extract().response();
 
         Assertions.assertEquals(200, response.getStatusCode());
+        System.out.println("Response body: " + response.getBody().asString());
         // Add additional assertions to validate the response body or structure
     }
```
Comment (sumukh-gn) [inline src/test/java/com/emart/AdminControllerTest.java:118]: please add annotation "@Deprecated" for the methods currently not in use

File: src/test/java/com/emart/AdminControllerTest.java [hunk 3]
```diff
@@ -22,71 +24,95 @@ public static void setup() {
...
+        int statusCode = response.getStatusCode();
+        String responseBody = response.getBody().asString();
+        System.out.println("Status code: " + statusCode);
+        System.out.println("Response body: " + responseBody);
+
+        Assertions.assertEquals(200, statusCode);
+//        Assertions.assertEquals("Admin Retrived successfully", responseBody);
     }
 
+
```
Comment (saurabh7nt) [inline src/test/java/com/emart/AdminControllerTest.java:44]: Please add Java guideline-wise comments and mention the description of all test cases.

File: src/test/java/com/emart/OrdersControllerTest.java [hunk 4]
```diff
@@ -70,7 +70,7 @@ public void testGetOrderById() {
```
Comment (suratkarmadhavi) [inline src/test/java/com/emart/OrdersControllerTest.java:70]: Comments not added.

File: src/main/java/com/emart/repository/CustomerRepository.java [hunk 5]
```diff
@@ -35,4 +35,8 @@ public interface CustomerRepository extends JpaRepository<Customer, Integer> {
     Optional<Object> getByUserName(@Param("user") String user);
     
     Customer findByUsername(String username);
+    
+    @Modifying
```
Comment (Devesh-Sharmaa) [inline src/main/java/com/emart/repository/CustomerRepository.java:39]: describe what this query is doing?

File: src/main/java/com/emart/controller/AdminController.java (hunk 1 above)
Comment (Shubhambal) [reply src/main/java/com/emart/controller/AdminController.java:76]: added the comments and rectified the code

File: src/main/java/com/emart/repository/CustomerRepository.java (hunk 5 above)
Comment (Shubhambal) [reply src/main/java/com/emart/repository/CustomerRepository.java:39]: thanks for addressing t the issue

File: src/test/java/com/emart/AdminControllerTest.java (hunk 3 above)
Comment (Shubhambal) [reply src/test/java/com/emart/AdminControllerTest.java:44]: okk i will do that

File: src/test/java/com/emart/AdminControllerTest.java (hunk 2 above)
Comment (Shubhambal) [reply src/test/java/com/emart/AdminControllerTest.java:118]: okk

**Review by sumukh-gn — APPROVED** [review]
okk you can proceed with the merge

**Review by saurabh7nt — APPROVED** [review]
i checked the changes that you made its is okk so now you can proceed with the merging

**Review by Devesh-Sharmaa — APPROVED** [review]
you resolved the issues  now you can proceed with the merge

**Review by suratkarmadhavi — APPROVED** [review]
issue is resolved you can proceed for merge

**Pull request merged**